test/bigtest/interactors/PurchaseOrder/open-order-confirmation-modal.js

```diff
@@ -0,0 +1,18 @@
+import {
+  interactor,
+  clickable,
+  isPresent,
+} from '@bigtest/interactor';
+
+const SUBMIT_BUTTON_SELECTOR = '[data-test-open-order-confirmation-modal-submit]';
+const CANCEL_BUTTON_SELECTOR = '[data-test-open-order-confirmation-modal-cancel]';
+const CONTENT_SELECTOR = '[data-test-open-order-confirmation-modal-content]';
+
+export default interactor(class OpenOrderConfirmationModalInct {
```

Comment:
`Inct` ? Too cryptic, IMHO, `OpenOrderConfirmationModal` is enough, it lives in interactors package